server/graphql.py

```diff
@@ -0,0 +1,36 @@
+from tartiflette import Resolver, Engine
+from tartiflette_starlette import TartifletteApp
+import os
+from .data import RECIPES, INGREDIENTS
+
+# Create a Tartiflette resolver for the `recipes` field.
+@Resolver("Query.recipes")
+async def resolve_recipes(parent, args, ctx, info):
+    return RECIPES
+
+
+# Create a Tartiflette resolver for the `recipe` field.
+@Resolver("Query.recipe")
+async def resolve_recipe(parent, args, ctx, info):
+    recipe = [r for r in RECIPES if r["id"] == int(args["id"])]
+
+    if not recipe:
+        return None
+
+    return recipe[0]
+
+
+# Create a Tartiflette resolver for the `ingredients` field,
+# which corresponds to a recipe.
+@Resolver("Recipe.ingredients")
+async def resolve_ingredients(parent, args, ctx, info):
+    if parent and parent["id"] in INGREDIENTS:
+        return INGREDIENTS[parent["id"]]
+
+    return None
+
+
+# Use schemas defined in the sdl directory.
+# See: https://tartiflette.io/docs/api/engine
+sdl = os.path.dirname(os.path.abspath(__file__)) + "/sdl"
+app = TartifletteApp(sdl=sdl, path="")
```

Comment:
So `path=""` is necessary here? The default is `"/"` but if it's forcing peeps to use this then we can definitely change it to `""`.

Reply:
Yes it is necessary

Reply:
OK, I think this is a flaw in `tartiflette-starlette`. The default `path` should allow to mount without issues (because I guess `/` then requires to access `/graphql/`).

Reply:
@cs01 I tried this out with Starlette, and it works out just fine since Starlette auto-completes trailing slashes. I think it's a Bocadillo issue!